lib/galaxy/util/checkers.py

```diff
@@ -14,6 +14,8 @@
 def check_html( file_path, chunk=None ):
     if chunk is None:
         temp = open( file_path, "U" )
+    elif hasattr(chunk, "splitlines"):
+        temp = chunk.splitlines()
```

Comment:
We already have the string in memory at this point - so this will only cause another several Kb of memory and it is all native so speed isn't too much of a concern. I was thinking I could rewrite this to use the iterator and control the line read size as well when I realized this piece of code really isn't serving any good purpose - and is causing problems such as preventing Notebooks from being reused. I'd rather revert the fix than spend more time on it at this point 😅.